src/main/webapp/WEB-INF/view/logout.jsp

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Login</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+  <nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/conversations">Conversations</a>
+    <% if(request.getSession().getAttribute("user") != null){ %>
+      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+    <% } else{ %>
+      <a href="/login">Login</a>
+    <% } %>
+    <a href="/about.jsp">About</a>
+    <% if(request.getSession().getAttribute("user") != null){ %>
```

Comment:
Will need to add a tab for All Users to be consistent with my code. Either you or I can do this depending on who merges their code first.

Reply:
I LOVE THIS. I wouldn't have been able to pick this out :)

Reply:
I'm not sure waht you mean by "All Users"